patches/api/0471-Fix-equipment-slot-and-group-API.patch

```diff
@@ -17,33 +21,55 @@ index 90f18f355a6a236a7e4273cc1258e7c8034b8276..a0750d457a9cd7cf3ab9c45bb28ce5a2
      public EquipmentSlot getSlot() {
          return slot == EquipmentSlotGroup.ANY ? null : slot.getExample();
      }
+diff --git a/src/main/java/org/bukkit/entity/LivingEntity.java b/src/main/java/org/bukkit/entity/LivingEntity.java
+index 5c29956c6db53440322330ff723c7087193641f1..a3e9360168a999fb08797285ecaec6578687676c 100644
+--- a/src/main/java/org/bukkit/entity/LivingEntity.java
++++ b/src/main/java/org/bukkit/entity/LivingEntity.java
+@@ -1447,4 +1447,14 @@ public interface LivingEntity extends Attributable, Damageable, ProjectileSource
+      */
+     void setBodyYaw(float bodyYaw);
+     // Paper end - body yaw API
++
++    // Paper start - Expose canUseSlot
++    /**
++     * Checks whether this entity can use the equipment slot
```

Comment:
An example or two would be good, like specifically calling out the body slot regarding players

Reply:
Added example linking to body slot, and body slot has javadoc explaining that only certain entities can have it